Seminar1/Task4.py

```diff
@@ -0,0 +1,21 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+# самое топорное решение
+
+sum = int(input('Сколько всего сделано журавликов: '))
+x = int(sum/6)
```

Comment:
Тут можно использовать целочисленное деление и не использовать функцию `int`
```suggestion
x = sum // 6
```